src/Groups/beta/custom/NewMgBetaGroupMember_Create.cs

```diff
@@ -43,7 +43,15 @@ public partial class NewMgBetaGroupMember_Create : System.Management.Automation.
 
         /// <summary>The reference to the client API class.</summary>
         public Groups Client => Module.Instance.ClientAPI;
+        /// <summary>Backing field for <see cref="CustomHeader" /> property.</summary>
+        private string _customHeader;
 
+        /// <summary>
+        /// CustomHeader Parameter. This should have a key:value and comma separated for multiple key:value pairs
+        /// </summary>
+        [System.Management.Automation.Parameter(Mandatory = false, HelpMessage = "CustomHeader Parameter. This should have a key:value and comma separated for multiple key:value pairs", ValueFromPipeline = true)]
+        [Category(ParameterCategory.Runtime)]
+        public string CustomHeader { get => this._customHeader; set => this._customHeader = value; }
```

Comment:
Can we:
- [ ] Change the `CustomHeader` type to a `IDictionary`. This will allow customers to use familiar PowerShell syntax when setting headers. See [Invoke-MgGraphRequest](https://github.com/microsoftgraph/msgraph-sdk-powershell/blob/a00ce42fc41d6c196e18e14d2649857f3feb41c4/src/Authentication/Authentication/Cmdlets/InvokeMgGraphRequest.cs#L72) implementation and [invoke-restmethod](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/invoke-restmethod?view=powershell-7.4#example-5-pass-multiple-headers) example.
- [ ] Rename `CustomHeader` parameters to `CustomHeaders` since customers can set more than one header value. This will align with the existing naming convention in `Invoke-MgGraphRequest` and other HTTP based commands.

The suggestions above will need to be applied to the AutoREST fork as well.

Reply:
The final experience should something like:
``` Powershell
# Setting a single header.
Get-MgUser -UserId $userId -CustomHeaders @{ 'Accept-Language' = 'en' }

# Setting multiple headers (option 1).
Get-MgUser -UserId $userId -CustomHeaders @{ 'Accept-Language' = 'en'; 'MySpecialHeader' = 'val1' }

# Setting multiple headers (option 2)
$MyCustomHeaders = @{
   'Accept-Language' = 'en'
   'MySpecialHeader' = 'val1'
}
Get-MgUser -UserId $userId -CustomHeaders $MyCustomHeaders 
```

We should also handle cases where duplicate headers are set.

Reply:
@calebkiage @sebastienlevert @peombwa thanks for the suggestion I will go ahead and implement the changes suggested by Peter

Reply:
@timayabi2020, @peombwa I still think the name should be `Headers` and not `CustomHeaders`, the `Invoke-WebRequest` cmdlet also uses that naming and it's shorter to type.

Reply:
@sebastienlevert, what do you think?

Reply:
@calebkiage we are using ``CustomHeaders`` which communicates that there is an option for customers to provide additional information that may be pertinent to them.  It will just append the information to the already existing default headers sent to graph APIs.

Reply:
In all other SDKs we use Headers in our request configurations to append to existing ones set by the underlying platform. +1 to naming it Headers.

Reply:
![image](https://github.com/microsoftgraph/msgraph-sdk-powershell/assets/10947120/28e75ed5-5ed3-4a14-a677-a18add97d5c8)

Reply:
Ok ``Headers`` it is.
![image](https://github.com/microsoftgraph/msgraph-sdk-powershell/assets/10947120/5e2a2fcf-ab50-4122-b629-59afb2ab857f)